Source/core/FileSystem.h

```diff
@@ -311,10 +317,11 @@ namespace Core {
             return (Create((USER_READ|USER_WRITE|USER_EXECUTE|GROUP_READ|GROUP_EXECUTE), exclusive));
         }
 
-        bool Create(const uint32_t mode, const bool exclusive = false)
+        bool Create(const uint32_t mode, const bool exclusive = false, bool large = false)
```

Comment:
What happens if by default we add this O_LARGEFILE flag and it is only a small file ?

Reply:
In 64bit machine, it will be anyway supporting large file if we pass FILE_OFFSET_BITS=64. But in 32-bit case we want to specify this flag to open a file if want more than 2GB size, but it any way create overheads with size and mapping. So I kept this as default param with false and can set whenever it is really required.

Reply:
Reverted back to the original code